storage/innobase/ddl/ddl0builder.cc

```diff
@@ -1961,11 +1961,13 @@ dberr_t Builder::finalize() noexcept {
   if (err == DB_SUCCESS) {
     write_redo(m_index);
 
-    DEBUG_SYNC_C_IF_THD(m_ctx.thd(), "row_log_apply_before");
+    if (apply_log) {
+      DEBUG_SYNC_C_IF_THD(m_ctx.thd(), "row_log_apply_before");
 
-    err = row_log_apply(m_ctx.m_trx, m_index, m_ctx.m_table, m_local_stage);
+      err = row_log_apply(m_ctx.m_trx, m_index, m_ctx.m_table, m_local_stage);
```

Comment:
is there a tab before err?

Reply:
no only spaces.

Reply:
I see a hightighted space in github, please check

Reply:
I think this just tells the difference of old code.  I have checked it is all spaces and not tab.

Reply:
ok